src/vhdl_backend.cc

```diff
@@ -844,6 +845,8 @@ bool dump_cell_expr(std::ostream &f, std::string indent, RTLIL::Cell *cell)
 	}
 
 	// TODO: port $eqx and $nex ("=" and "/=" return BIT, not STD_LOGIC)
+	// TODO: ?= and friends may be a VHDL-2008 addition?
```

Comment:
Yes, they are.

Reply:
Small code adjustment not to use `?=` when not in VHDL-2008 mode; a full fix requires finding a way to backport this operator, which is deferred until later.

Reply:
I might be wrong, but I think that these operators are just synctactic sugar. The purpose is to avoid having to convert booleans to std_logic.

```vhdl
x <= a?=b;
```

```vhdl
x <= '1' when a=b else '0';
```

/cc @Paebbels @JimLewis for confirmation.